hasura.planx.uk/migrations/1734623060683_create_table_public_temp_data_migrations_audit/up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE "public"."temp_data_migrations_audit" (
+  "flow_id" uuid NOT NULL, 
+  "team_id" integer NOT NULL, 
```

Comment:
Would relationships here be helpful for queries in the migration scripts?

Reply:
I don't think so ! As I see it we'll simply return the `flow_id` from this table (it's the primary key so indexed) then pass into a GraphQL relational query which will fetch the flow and its' latest published version (so meaningful foreign key is at GraphQL & existing `flows` table level rather than on this one?). But please object if I'm missing a benefit of fkeys directly here!

Reply:
Oof slow brain today - answered my own question there - you're totally right, if we set a proper fkey to flows here then migration script will be able to make a single query rather than two :+1: update incoming!

Reply:
:white_check_mark: 
```graphql
query MyQuery {
  temp_data_migrations_audit(where: {updated: {_eq: false}}, limit: 1) {
    flow_id
    flows {
      data
      published_flows(order_by: {created_at: desc}, limit: 1) {
        data
      }
    }
  }
}
```